ros/src/pybullet_ros/plugins/control.py

```diff
@@ -5,26 +5,27 @@
 """
 
 import rospy
-from std_msgs.msg import Float64
+from std_msgs.msg import Float64MultiArray
+
 
 # NOTE: 2 classes are implemented here, scroll down to the next class (Control) to see the plugin!
 
+
 class pveControl:
     """helper class to receive position, velocity or effort (pve) control commands"""
-    def __init__(self, joint_index, joint_name, controller_type):
+
+    def __init__(self, controller_type):
         """constructor
         Assumes joint_name is unique, creates multiple subscribers to receive commands
         joint_index - stores an integer joint identifier
         joint_name - string with the name of the joint as described in urdf model
         controller_type - position, velocity or effort
         """
-        assert(controller_type in ['position', 'velocity', 'effort'])
-        rospy.Subscriber(joint_name + '_' + controller_type + '_controller/command',
-                         Float64, self.pve_controlCB, queue_size=1)
+        assert (controller_type in ['position', 'velocity', 'effort'])
```

Comment:
remove `( )` from this line, since `assert` is a keyword statement and not a function (same as `if`, `with`...)
```suggestion
        assert controller_type in ['position', 'velocity', 'effort']
```